meson.build

```diff
@@ -269,6 +269,12 @@ if get_option('plugin_thunderbolt')
   conf.set('HAVE_THUNDERBOLT', '1')
 endif
 
+if get_option('plugin_flashrom')
```

Comment:
You'll need to add this same check to `./plugins/meson.build` to only build this plugin when this configuration option is found.